crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -119,31 +120,31 @@ fn init_client(cfg: &Config) -> Result<()> {
     wait_for_node_to_initialize(cfg)?;
 
     // 2. Wait for the node to be bootstrapped
-    info!("Waiting for node to bootstrap...");
-    cfg.octez_client()?.wait_for_node_to_bootstrap()?;
-    info!(" done");
+    print!("Waiting for node to bootstrap...");
+    cfg.octez_client_sandbox()?.wait_for_node_to_bootstrap()?;
+    println!(" done");
 
```

Comment:
Please don't revert the logging changes (ditto else where). After rebasing on `main`, these should be `debug!`s